gradle.properties

```diff
@@ -7,10 +7,12 @@ minecraft_version=1.13.2
 # Testing: net.minecraftforge.test:forge
 # Normal:  net.minecraftforge:forge
 forge_base=net.minecraftforge:forge
-forge_version=25.0.54
+forge_major=25.0
+forge_minor=63
 
 # Mod specific
 version=2.99.0
+api_version=0.0.0
```

Comment:
0.1.0 or start at 1.0.0 as thats what'll be released? 0.0.0 makes no sense haha

Reply:
I put it to 0.0.0, as currently there is no API at all - once there is at least something I thought we'd then bump it up to an appropiate number